gitlint/tests/test_body_rules.py

```diff
@@ -70,6 +70,12 @@ def test_body_first_line_empty(self):
         violations = rule.validate(commit)
         self.assertListEqual(violations, [expected_violation])
 
+        # Skip body checks, if regex matches
+        rules.BodyMinLength({'ignore-body-checks-regex': 'Release\s([a-z\-])*\s(\d+\.)?(\d+\.)?(\*|\d+)'})
+        commit = self.gitcommit(u"Release foo-bar-baz 0.12.3")
+        violations = rule.validate(commit)
+        self.assertIsNone(violations)
+
```

Comment:
That test should go into a separate test method.

Reply:
Hmm, sounds like harassment, but Done

Reply:
The benefit of separate test cases is that it easier to spot which part breaks and it prevents hiding failures (the test will abort on the first assertion error).